src/constants/types.js

```diff
@@ -57,6 +57,11 @@ export const SUGGESTIONS_STATUS = [
   },
 ];
 
+export const MODAL_TYPE = {
```

Comment:
i would rename to 
`modalType={
  ADD:'add',
  EDIT:'EDIT'
}`

Reply:
Im not sure because I already have all my types defined this way, with uppercase the name of the type and camelCase the name inside the object 😅